cmd/containerboot/main_test.go

```diff
@@ -68,6 +70,8 @@ func TestContainerBoot(t *testing.T) {
 	if err != nil {
 		t.Fatalf("error unmarshaling egress services config: %v", err)
 	}
+	// same as egressSvcCfg, but base64 encoded.
+	egressSvcsCfgBase64 := "eyJwb2RJUHY0IjoiIiwic2VydmljZXMiOnsiZm9vIjp7InBvcnRzIjpbXSwidGFpbG5ldFRhcmdldElQcyI6bnVsbCwidGFpbG5ldFRhcmdldCI6eyJpcCI6IiIsImZxZG4iOiJmb28udGFpbG5ldHh5eC50cy5uZXQifX19fQ=="
```

Comment:
nit: for easier maintainability would it be possible to write this string in non-base64 and encode it during the test?